### PR TITLE
Fixed  views and agents bugs

### PR DESCRIPTION
**Pull request by moneya**

### CODE REVIEW

File: app/Livewire/Agents/Partials/Card.php [hunk 1]
```diff
@@ -11,14 +12,26 @@ class Card extends Component
 
     public $agent;
 
-    public function mount($selectedAgent)
+    public function mount($se7lectedAgent)
```
Comment (svemir) [inline app/Livewire/Agents/Partials/Card.php]: This looks like a typo? Probably should remain `$selectedAgent`

File: app/Livewire/Chat.php [hunk 2]
```diff
@@ -73,7 +72,7 @@ public function mount($id = null)
         // If ID is not null, we're in a thread. But if thread doesn't exist or doesn't belong to the user and doesn't match the session ID, redirect to homepage.
         if ($id) {
             $thread = Thread::find($id);
-            if (! $thread || (auth()->check() && $thread->user_id !== auth()->id()) || (! auth()->check() && $thread->session_id !== session()->getId())) {
+            if (!$thread || (auth()->check() && $thread->user_id !== auth()->id()) || (!auth()->check() && $thread->session_id !== session()->getId())) {
```
Comment (svemir) [inline app/Livewire/Chat.php]: It also seems that `./vendor/bin/pint` was not run, or it ran with wrong parameters, so we lost all spaces after the negations.

**Review by svemir — CHANGES_REQUESTED** [review]

File: app/Livewire/Chat.php (hunk 2 above)
Comment (moneya) [reply app/Livewire/Chat.php]: It ran but maybe with the config.
This is still a WIP

**Comment by riccardobl** [comment]
Please fix this PR as it is currently reverting my last commits

**Pull request merged**